src/lib/utils.ts

```diff
@@ -1,9 +1,10 @@
 import { fromUtf8, toHex, toUtf8 } from '@cosmjs/encoding';
 import { BroadcastTxFailure, Coin, logs, StdFee } from '@cosmjs/stargate';
+import { parseEvent } from '@cosmjs/stargate/build/logs';
```

Comment:
Importing from the build dir smells funny.
This is [kind of exposed top-level](https://github.com/cosmos/cosmjs/blob/79396bfaa49831127ccbbbfdbb1185df14230c63/packages/stargate/src/index.ts#L55)

I think you need :

```ts
import { logs } from '@cosmjs/stargate';
const { parseEvent } = logs;
```